### PR TITLE
FIX: Refactor experimental setup now with permanent installation

### LINKED ISSUE #117: Should the ECG cable inside the scanning room get away from the MRI perpendicularly?
See the conversation started at https://github.com/TheAxonLab/hcph-sops/pull/116/files#r1307960763

### LINKED ISSUE #129: Update SOPs with O2 concentration recording
We are about to receive the new cable...

### LINKED ISSUE #139: [INSERT PICTURE] and [TOCHECK] in pre-session.md
They are right above the [setting up the projector](https://www.axonlab.org/hcph-sops/data-collection/pre-session/#setting-up-the-projector) section.

### LINKED ISSUE #172: Move most of the device setup from pre-session to preliminary
Thanks to Eleonora, there are now some shelves for devices, and we can have the BIOPAC et al. there during the duration of the acquisition efforts. THANKS ELEONORA!

### PR DESCRIPTION
**Pull request by oesteban**
This PR updates all our protocols to the new setup and cleans up some incorrect stuff.

Resolves: #172.
Resolves: #139.
Resolves: #129.
Resolves: #117.
Relates: #120.

### CODE REVIEW

File: docs/data-collection/intro.md [hunk 1]
```diff
@@ -54,16 +54,17 @@ The above graph can be broken down as follows:
...
-
 1. **Eye tracker (ET)**.
-    The ET is composed of two main elements:
-    (i) inside the scanner's bore, we place an arm that holds an infrared lens and camera sensor on one side and an infrared lamp that illuminates the right eye of the subject through a special mirror to reflect the infrared spectrum; and (ii) a PC tower that receives the camera recordings, postprocess the images and calculates the final parameters of interest (position of the eye, pupil size, etc.).
+    We use the *{{ settings.eyetracker.model }}* ({{ settings.eyetracker.vendor }}, {{ settings.eyetracker.place }}).
+    Our particular variant «{{ settings.eyetracker.submodel }}» is composed of three main elements:
+    (i) inside the scanner's bore, we place an arm that holds an infrared lens and camera sensor on one side and an infrared lamp that illuminates the {{ settings.eyetracker.eye }} eye of the subject through a special mirror (ii) to reflect the infrared spectrum; and (iii) a PC tower that receives the camera recordings, postprocesses the images and calculates the final parameters of interest (position of the eye, pupil size, etc.).
```
Comment (acionca) [inline docs/data-collection/intro.md]: I feel like it is clearer that way
```suggestion
    (i) inside the scanner's bore, we place an arm that holds an infrared lens and camera sensor on one side and an infrared lamp that illuminates the {{ settings.eyetracker.eye }} eye of the subject through (ii) a special mirror to reflect the infrared spectrum; and (iii) a PC tower that receives the camera recordings, postprocesses the images and calculates the final parameters of interest (position of the eye, pupil size, etc.).
```

**Review by esavary — COMMENTED** [review]
It's not within the modified lines, but I believe 'including gas contents with a gas analyzer' should be replaced with 'including expelled gas concentration or something similar in the first paragraph of intro.md

File: docs/data-collection/legacy-et.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+**Placing the infrared mirror**
+
+- [ ] Detach the standard mirror's frame from the head coil, if it is placed there.
+    Remove unnecessary items from the scanning bed, and prepare the mirror to attach the infrared mirror of the ET at a later step.
+- [ ] Prepare two long strips of scotch tape and leave them in a convenient place to attach the ET mirror later.
```
Comment (acionca) [inline docs/data-collection/legacy-et.md]:
```suggestion
- [ ] Prepare two long strips of tape and leave them in a convenient place to attach the ET mirror later.
```

File: docs/data-collection/participant-prep.md [hunk 3]
```diff
@@ -219,7 +219,10 @@
 ### Connecting physiological recording sensors and probes
 - [ ] Connect the ECG leads to the three electrodes. The electrodes MUST be connected following the color scheme shown in the picture below.
     ![prep-ecg-electrodes](../assets/images/prep-ECG-electrodes.png)
-- [ ] Install the RB below the participant's chest and connect it to the tube as shown in the picture below. The RB measures the stretching induced by breathing, so it MUST surround the chest or stomach comfortably. Positioning the RB higher (chest) or lower (stomach) depends on the individual's preferential respiration mode (chest breathing or diaphragmatic, respectively).
+- [ ] Install the RB below the participant's chest and connect it to the tube as shown in the picture below.
+    The RB measures the stretching induced by breathing, so it MUST surround the chest or stomach comfortably. Positioning the RB higher (chest) or lower (stomach) depends on the individual's preferential respiration mode (chest breathing or diaphragmatic, respectively).
+- [ ] Connect the distal end of the RB tube to the RB and leave it on the bed.
+    ![RB_connection](../assets/images/RB_connection.jpg "RB_connection")
```
Comment (acionca) [inline docs/data-collection/participant-prep.md]: I think there's a mistake here. Should the RB be on the participant ? or on the bed ?
```suggestion
- [ ] Install the RB below the participant's chest and connect it to the tube as shown in the picture below.
    The RB measures the stretching induced by breathing, so it MUST surround the chest or stomach comfortably.
    Positioning the RB higher (chest) or lower (stomach) depends on the individual's preferential respiration mode (chest breathing or diaphragmatic, respectively).
    ![RB_connection](../assets/images/RB_connection.jpg "RB_connection")
```

File: docs/data-collection/participant-prep.md [hunk 4]
```diff
@@ -219,7 +219,10 @@
 ### Connecting physiological recording sensors and probes
 - [ ] Connect the ECG leads to the three electrodes. The electrodes MUST be connected following the color scheme shown in the picture below.
     ![prep-ecg-electrodes](../assets/images/prep-ECG-electrodes.png)
-- [ ] Install the RB below the participant's chest and connect it to the tube as shown in the picture below. The RB measures the stretching induced by breathing, so it MUST surround the chest or stomach comfortably. Positioning the RB higher (chest) or lower (stomach) depends on the individual's preferential respiration mode (chest breathing or diaphragmatic, respectively).
+- [ ] Install the RB below the participant's chest and connect it to the tube as shown in the picture below.
+    The RB measures the stretching induced by breathing, so it MUST surround the chest or stomach comfortably. Positioning the RB higher (chest) or lower (stomach) depends on the individual's preferential respiration mode (chest breathing or diaphragmatic, respectively).
+- [ ] Connect the distal end of the RB tube to the RB and leave it on the bed.
```
Comment (esavary) [inline docs/data-collection/participant-prep.md]: Should we mention disconnecting for calibration here?

Comment (oesteban) [reply docs/data-collection/participant-prep.md]: That should be said in `scanning.md`

**Comment by oesteban** [comment]
> It's not within the modified lines, but I believe 'including gas contents with a gas analyzer' should be replaced with 'including expelled gas concentration or something similar in the first paragraph of intro.md

Sure, let's do that on a separate PR.

File: docs/data-collection/scanning-notes.md [hunk 5]
```diff
@@ -41,32 +41,64 @@
...
-    !!! warning "If the two crosses do not appear, the coverage, focus and intensity of the ET are incorrect"
+        When assigning names to the MR sequences in the protocol, make sure to follow the [Reproin conventions](https://dbic-handbook.readthedocs.io/en/latest/mri/reproin.html) to maximally facilitate the conversion into BIDS.
 
-        - [ ] Repeat the steps for their setting up given in [the participant's preparation section](participant-prep.md#final-preparatory-steps-of-the-et)
+- [ ] Update the *Number of measurements* in all `func-bold_task-*` sequences, according to the [previously recorded timings](intro.md#task-timing):
 
-- [ ] Initiate the ET calibration by pressing <span class="keypress">C</span> on the laptop keyboard or by clicking on <span class="keypress">Calibration</span> on the ET interface.
-- [ ] Once the participant's gaze stabilizes on the first fixation point, the <span class="keypress">Accept Fixation</span> button turns green.
-    Click on it to confirm the initial position.
+    $$N_\text{measurements} = L_t / \text{TR}, \quad t \in \{\text{bht}, \text{qct}, \text{rest}\},$$
```
Comment (acionca) [inline docs/data-collection/scanning-notes.md:56]: This did not rendered properly on my side – I had to to the following to make it render:
```suggestion
    $N_\text{measurements} = L_t / \text{TR}, \quad t \in \{\text{bht}, \text{qct}, \text{rest}\},$
```

**Review by esavary — COMMENTED** [review]
In 'presession.md' , regarding 'The OK button MUST be clicked within this modal dialog on the secondary screen,' it's also important to keep the mouse on the second screen until it has finished measuring the frame rate.

**Review by acionca — APPROVED** [review]

File: docs/data-collection/scanning-notes.md (hunk 5 above)
Comment (oesteban) [reply docs/data-collection/scanning-notes.md:56]: That's weird - the double $ should make the equation in a new block (as opposed to the single $ that writes it inline).

I see it properly rendered on https://www.axonlab.org/hcph-sops/data-collection/preliminary/ and locally, so I'll skip this suggestion and leave it for you to investigate further - some plugin or addon must be missing on your settings?

**Pull request merged**